scripts/after_platform_add.js

```diff
@@ -0,0 +1,25 @@
+var async = require('async');
+var spawn = require('./spawn');
+
+var addPlugin = function(plugin, callback) {
+    spawn('cordova', ['plugin', 'add', plugin], callback);
+};
+
+module.exports = function(context) {
+    var Q = context.requireCordovaModule('q');
```

Comment:
yeah, it's the cordova recommended way of having async hooks... and it's freaking weird